PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListItemCell.swift

```diff
@@ -275,6 +276,10 @@ extension ItemsListItemCell {
         let menuActions = state.model?.overflowActions.compactMap(UIAction.init) ?? []
         menuButton.menu = UIMenu(children: menuActions)
 
+        if let trackAction = state.model?.trackOverflow {
+            menuButton.addAction(trackAction, for: .menuActionTriggered)
+        }
```

Comment:
Is this a replacement for calling `trackButton` on all the overflow actions themselves? Kind of a catch-all place.

Reply:
this is to detect when the button is tapped on, it seems that the overflow actions function is called even though the user has not tapped on it because we access it above in line 276